src/service/OpsLevelController.ts

```diff
@@ -86,7 +86,7 @@ export class OpsLevelController {
   public async getAutoSyncConfiguration() {
     return {
       "auto_sync_enabled": (await this.db.fetchConfigValue("auto_sync_enabled") === "true"),
-      "auto_sync_schedule": await this.db.fetchConfigValue("auto_sync_schedule") || "",
+      "auto_sync_schedule": await this.db.fetchConfigValue("auto_sync_schedule") || "0 * * * *",
```

Comment:
Presumably, customers won't even know an expert is happening. Should we make the default less frequent? Once per day or week.

Reply:
the default is also enabled=false, I don't imagine they would be able to turn it on without seeing the schedule right below